nevergrad/benchmark/frozenexperiments.py

```diff
@@ -9,13 +9,53 @@
 from nevergrad.optimization.base import ConfiguredOptimizer
 from nevergrad.optimization import experimentalvariants  # pylint: disable=unused-import
 from nevergrad.functions import ArtificialFunction
+from nevergrad.functions.perfcap3d import core as perfcap3d
```

Comment:
```suggestion
from nevergrad.functions import perfcap3d
```
then in `perfcap.__init__.py` do: 
```python
from core import Perfcap3DServerExecutor as Perfcap3DServerExecutor
from core import Perfcap3DFunction as Perfcap3DFunction
```